src/main/java/seedu/duke/command/ChooseVenueCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.duke.command;
+
+import seedu.duke.event.Event;
+import seedu.duke.exception.InvalidIndexException;
+import seedu.duke.venue.VenueList;
+
+public class ChooseVenueCommand extends Command{
+
+    protected int venueNum;
+
+    public ChooseVenueCommand(String commandType, int venueNum) {
+        super(commandType);
+        this.venueNum = venueNum;
+    }
+
+    @Override
+    public void execute(Event event, VenueList venueList) {
+        try{
+            event.updateVenue(venueList, venueNum);
+        } catch (InvalidIndexException err){
```

Comment:
perhaps a space can be added between `)` and `{` for the codestyle to be consistent throughout